tests/UNMOCKED_end2end_tests/test_al_step3.py

```diff
@@ -0,0 +1,78 @@
+import tempfile
+import os
+import time
+
+import numpy as np
+
+from lightly.active_learning.agents.agent import ActiveLearningAgent
+from lightly.active_learning.scorers.classification import ScorerClassification
+from lightly.api.api_workflow_client import ApiWorkflowClient
+from lightly.api.bitmask import BitMask
+from lightly.data import LightlyDataset
+from lightly.api.upload import upload_dataset
+from lightly.openapi_generated.swagger_client import SamplingMethod
+
+from lightly.utils import save_embeddings
+
+from lightly.active_learning.config.sampler_config import SamplerConfig
+
+
+def t_est_unmocked_complete_workflow(path_to_dataset: str, token: str, dataset_id: str):
+    # define the api_client and api_workflow
+    api_workflow_client = ApiWorkflowClient(host="https://api-dev.lightly.ai", token=token, dataset_id=dataset_id)
```

Comment:
Can you switch to getting the host from the env variable? Also in the other unmocked script because I don't feel so comfortable exposing our developer api to everyone.

Reply:
I would not call it "exposing" the api, as it will be accessible from everywhere even if we do not give the easily-to-guess URL in the pip package. I someone really wants to get to it, trying out "api-dev" is not really harder than searching our pip-package for some URLs. The URL is protected by a token anyways. In my eyes removing the url is more like preventing burglary on your house by removing your address from the telephone book.
If you really want me to, I could change that, but I do not see it as a security measure.

Reply:
I don't see it as a security threat, more as a bad practice but focus on the important stuff first

Reply:
I would keep it there as it means one less error which could be made while testing.